File: jdroid-android-google-inappbilling/src/debug/java/com/jdroid/android/google/inappbilling/TestDeveloperPayloadVerificationStrategy.java
package com.jdroid.android.google.inappbilling;

import com.jdroid.android.google.inappbilling.client.DeveloperPayloadVerificationStrategy;
import com.jdroid.android.google.inappbilling.client.Product;

public class TestDeveloperPayloadVerificationStrategy implements DeveloperPayloadVerificationStrategy {
	
	public static final String VALID_TEST_VERIFICATION_ENABLED = "validTestVerificationEnabled";
	
	@Override
	public Boolean verify(Product product) {
		return true;
	}
}

File: jdroid-android-google-inappbilling/src/debug/java/com/jdroid/android/google/inappbilling/TestDeveloperPayloadVerificationStrategy.kt
package com.jdroid.android.google.inappbilling

import com.jdroid.android.google.inappbilling.client.DeveloperPayloadVerificationStrategy
import com.jdroid.android.google.inappbilling.client.Product

class TestDeveloperPayloadVerificationStrategy : DeveloperPayloadVerificationStrategy {

    override fun verify(product: Product): Boolean {
        return true
    }

    companion object {
        const val VALID_TEST_VERIFICATION_ENABLED = "validTestVerificationEnabled"
    }
}
